src/Slist.hs

```diff
@@ -490,6 +491,21 @@ cycle sl@(Slist _ Infinity) = sl
 cycle Slist{..}             = infiniteSlist $ L.cycle sList
 {-# INLINE cycle #-}
 
+{- | @O(1)@. A lazy slist equivalent of Haskell's '[from..to]' notation:
```

Comment:
It's also an equivalent to `P.enumFromTo`. Also a minor Haddock error:

```suggestion
{- | @O(1)@. A slist equivalent of 'P.enumFromTo' function or @[from..to]@ notation:
```
I removed the word "lazy" as the behaviour is the same as with the `[a..b]`.